codec/console/common/src/read_config.cpp

```diff
@@ -37,8 +37,9 @@
  *      08/18/2008 Created
  *
  *****************************************************************************/
-
+#if !defined(_CRT_SECURE_NO_WARNINGS)
```

Comment:
This change (and the other similar one) is entirely unrelated to the rest of this pull request. Please refrain from doing unrelated changes in one pull request; separate such changes out to a separate on if they are needed.

Reply:
I see.
Thanks for pointing out..